SANIS-1 doc-step-card

pbokovtakeoffstaff opened the pull request:
- Создание 3х карточек
- Выстраивание по цетру
- Добавление глобал стиля

pbokovtakeoffstaff committed:
Проба пера

Не получилось решить проблему scss in css

pbokovtakeoffstaff committed:
Проба пера

Не получилось решить проблему scss in css

pbokovtakeoffstaff committed:
Проба пера

Удалось настроить scss in css

pbokovtakeoffstaff committed:
Добил все по Ридми

pbokovtakeoffstaff committed:
Merge branch 'master' into BokovPV-BEM

# Conflicts:
#	README.md

pbokovtakeoffstaff committed:
Обновил Редми

pbokovtakeoffstaff committed:
First button

pbokovtakeoffstaff committed:
Первые кнопки.

- Создание кнопок
- Добавление стилей Элементов
- Добавление стилей модификаторов
- Hover

sergamers committed:
Создание компонента Icon

sergamers committed:
Update README.md

pbokovtakeoffstaff committed:
Merge remote-tracking branch 'origin/master' into BokovPV-BEM

# Conflicts:
#	.gitignore
#	README.md
#	package-lock.json
#	package.json

pbokovtakeoffstaff committed:
Установка normalize.css

pbokovtakeoffstaff committed:
Установка normalize.css

pbokovtakeoffstaff committed:
Первая кнопочка макета

pbokovtakeoffstaff committed:
Первая кнопочка макета

pbokovtakeoffstaff committed:
Merge branch 'lesson-2' into BokovPV-BEM

sergamers committed:
Update README.md

pbokovtakeoffstaff committed:
Merge branch 'lesson-2' into BokovPV-BEM

pbokovtakeoffstaff committed:
Смена путей

pbokovtakeoffstaff committed:
Пробы кнопок

sergamers committed:
мелкие правки

pbokovtakeoffstaff committed:
Пробы кнопок

pbokovtakeoffstaff committed:
Правка doc

sergamers committed:
Update README.md

sergamers committed:
Update README.md

pbokovtakeoffstaff committed:
Merge remote-tracking branch 'origin/lesson-2' into BokovPV-BEM

pbokovtakeoffstaff committed:
Item card

- Правки после проверки ДЗ
- Добавление itemCard + стили

pbokovtakeoffstaff committed:
Item card

- Убраны лишний пробелы

pbokovtakeoffstaff committed:
Item card

- Короткие правки

pbokovtakeoffstaff committed:
Item card

- Стили - text-align

pbokovtakeoffstaff committed:
Merge

pbokovtakeoffstaff committed:
doc-step-card

- Создание 3х карточек
- Выстраивание по цетру
- Добавление глобал стиля

pbokovtakeoffstaff committed:
doc-step-card

- Создание 3х карточек
- Выстраивание по цетру
- Добавление глобал стиля
- Удалил Папку src

pbokovtakeoffstaff committed:
new-item-card

- Создание одной карточки
ToDo - Решить проблему с флекс позиционированием

pbokovtakeoffstaff committed:
new-item-card

- Решил проблему с несколькими карточками, но мне все равно не нравится решение

pbokovtakeoffstaff committed:
new-item-card

- Решил проблему с несколькими карточками, но мне все равно не нравится решение
- Добавил ToDo

pbokovtakeoffstaff committed:
new-item-card

- Добавил переменные и нашел более лучшее решение по флексам через margin

sergamers commented on index.html:
```diff
@@ -0,0 +1,10 @@
+<body>
+
+</body>
+</html>
```
Удалить

pbokovtakeoffstaff replied:
Удалил

sergamers commented on package.json:
```diff
@@ -18,6 +18,8 @@
   "dependencies": {
-    "normalize.css": "^8.0.1"
+    "normalize.css": "^8.0.1",
+    "sass": "^1.56.2",
```
"sass": "^1.56.2", - удалить

pbokovtakeoffstaff replied:
Удалил

sergamers commented on bokov/src/components/doc-step-card/doc-step-card.css:
```diff
@@ -0,0 +1,29 @@
+    display: flex;
+    justify-content: space-between;
+    width: 1020px;
+
```
margin: auto

pbokovtakeoffstaff replied:
Исправил на auto

sergamers commented on bokov/src/components/doc-step-card/doc-step-card.html:
```diff
@@ -0,0 +1,70 @@
+</head>
+<body>
+
+<div class="BlockCenter">
```
Удалить блок .BlockCenter

pbokovtakeoffstaff replied:
Удален

sergamers commented on bokov/src/components/new-item-card/new-item-card.html:
```diff
@@ -0,0 +1,79 @@
+    <link rel="stylesheet" href="new-item-card.css">
+    <link rel="stylesheet" href="../label/label.css">
+    <link rel="stylesheet" href="../../global/global.css">
+    <script src="/node_modules/svg-to-inline/dist/svg-to-inline.js"></script>
```
удалить

pbokovtakeoffstaff replied:
Удален

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+    border-radius: 10px;
+    margin-left: 10px;
+
+}
```
margin-left вынести в другую сущность

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+
+}
+
+.Label {
```
превратить в элемент

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+.Label {
+    margin-top: 26px;
+    margin-left: var(--card-marginLeft);
+}
```
если больше 1 margin стороны, то лучше писать короткую конструкцию

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+    display: flex;
+    justify-content: center;
+    margin-top: 51px;
+}
```
Подумать над решение как bg

sergamers replied:
justify-content

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+    color: #000000;
+    margin-top: 15px;
+    margin-left: var(--card-marginLeft);
+}
```
удалить margin-left. Переписать на padding главного блока. Посмотреть в сторону box-sizing

sergamers commented on bokov/src/components/new-item-card/new-item-card.html:
```diff
@@ -0,0 +1,79 @@
+        </div>
+        <div class="ItemCard-describe">
+            Состав: белое золото 585 пробы, бриллианты
+        </div>
```
Рекомендую обрамить в еще один div ( см как изменится поведение justify-content )

pbokovtakeoffstaff replied:
Готово

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+}
+
+.ItemCard-text {
+    display: flex;
```
display: flex; зачем?

pbokovtakeoffstaff replied:
Убрал

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+}
+
+.ItemCard-describe {
+    display: flex;
```
display: flex; ?

pbokovtakeoffstaff replied:
Убрал

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,51 @@
+    display: flex;
+    width: 291px;
+    height: 26px;
+    font: 400 12px/15px Arial;
```
Подумать как удалить  width: и height:

pbokovtakeoffstaff replied:
Готово

sergamers commented on bokov/src/global/global.css:
```diff
@@ -0,0 +1,15 @@
+    display: flex;
+    justify-content: center;
+
+}
```
<style>
    .helper, .helper *{
        outline: solid 1px red;
    }
</style>

pbokovtakeoffstaff replied:
Применил

sergamers reviewed (commented):
ревью

pbokovtakeoffstaff committed:
new-item-card

- Сделаны правки по код ревью
- Добавлена картинка в bg-image + добавлен модификатор itemCard--icon
- Добавил doc

pbokovtakeoffstaff committed:
new-item-card

- Добавлены группы

pbokovtakeoffstaff committed:
new-item-card

- Добавлены группы

pbokovtakeoffstaff committed:
Merge branch 'lesson-3' into BokovPV

sergamers committed:
Merge branch 'lesson-3' into BokovPV

sergamers commented on bokov/src/components/new-item-card/new-item-card.html:
```diff
@@ -0,0 +1,85 @@
+    <title>new-item-card</title>
+    <link rel="stylesheet" href="new-item-card.css">
+    <link rel="stylesheet" href="../label/label.css">
+    <link rel="stylesheet" href="../../global/global.css">
```
удалить лишний линк глобал

pbokovtakeoffstaff replied:
Удалил

pbokovtakeoffstaff committed:
new-item-card

- удалил лишний link

pbokovtakeoffstaff committed:
Merge remote-tracking branch 'origin/BokovPV' into BokovPV

pbokovtakeoffstaff committed:
new-item-card

- Исправил нейминг блоков css

sergamers commented on bokov/src/components/new-item-card/new-item-card.html:
```diff
@@ -0,0 +1,85 @@
+
+<div class="helper ContainerItemCard">
+
+    <div class="ItemCard itemCard--icon1">
```
блоки начинаем писать с большой буквы

pbokovtakeoffstaff replied:
Поправил

sergamers commented on bokov/src/components/new-item-card/new-item-card.html:
```diff
@@ -0,0 +1,85 @@
+
+    <div class="ItemCard itemCard--icon1">
+
+        <div class="Label">
```
должен быть элементом

pbokovtakeoffstaff replied:
Label убран

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,71 @@
+}
+
+.Label {
+    margin-top: 3px;
```
убрать маржины из блока и переписать на паддинг в корневом блоке

pbokovtakeoffstaff replied:
Убрал Лейбл

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,71 @@
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-top: 15px;
```
марджин-топ далить

pbokovtakeoffstaff replied:
Убрал

pbokovtakeoffstaff committed:
new-item-card

- Убрал Label
- Удалил закомментированный код

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,71 @@
+.itemCard-text {
+    /*display: flex;*/
+    /*width: 251px;*/
+    /*height: 46px;*/
```
если используешь закоментированный код- то оставь коментарии для чего иначе удали

pbokovtakeoffstaff replied:
Убрал комменты

sergamers commented on bokov/src/components/new-item-card/new-item-card.css:
```diff
@@ -0,0 +1,71 @@
+    /*display: flex;*/
+    /*width: 251px;*/
+    /*height: 46px;*/
+    font: 500 18px/22px Arial;
```
добавь пожалуйста шрифт как на макете

pbokovtakeoffstaff replied:
Убрать комменты

pbokovtakeoffstaff committed:
new-item-card

- Убрал Label
- Удалил закомментированный код
- Добавил шрифт

pbokovtakeoffstaff committed:
new-item-card

- Поправил bg

sergamers committed:
Merge branch 'lesson-3' into BokovPV

sergamers committed:
review
